Failed to build mbedtls on install

**Describe the bug**
Following the [install doc](https://microsoft.github.io/CCF/main/build_apps/install_bin.html), I installed ansible and ran the `app-dev.yml` playbook. When building `mbedtls`, it fails with error:
```
make: *** No targets specified and no makefile found. Stop.
```

**To Reproduce**
```
cd getting_started/setup_vm/
./run.sh app-dev.yml
```

**Environment information**
Ubuntu 20.04.3
Intel W-2155 (no SGX)

**Additional context**
Full log:
```
  "changed": true,
  "cmd": "cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_POSITION_INDEPENDENT_CODE=ON .. make",
  "delta": "0:00:01.289875",
  "end": "2021-09-17 10:05:32.757398",
  "msg": "non-zero return code",
  "rc": 2,
  "start": "2021-09-17 10:05:31.467523",
  "stdout":
    -- The C compiler identification is GNU 9.3.0
    -- Check for working C compiler: /usr/bin/cc
    -- Check for working C compiler: /usr/bin/cc -- works
    -- Detecting C compiler ABI info
    -- Detecting C compiler ABI info - done
    -- Detecting C compile features
    -- Detecting C compile features - done
    -- Found PythonInterp: /usr/bin/python (found version "3.8.10") 
    -- Found Perl: /usr/bin/perl (found version "5.30.0") 
    -- Looking for pthread.h
    -- Looking for pthread.h - found
    -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
    -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
    -- Looking for pthread_create in pthreads
    -- Looking for pthread_create in pthreads - not found
    -- Looking for pthread_create in pthread
    -- Looking for pthread_create in pthread - found
    -- Found Threads: TRUE  
    -- Configuring done
    -- Generating done
    -- Build files have been written to: /tmp/mbedtls-mbedtls-2.16.10/build"
  "stderr":
    make: *** No targets specified and no makefile found. Stop.
```

--- a/getting_started/setup_vm/roles/ccf_build/tasks/install.yml
+++ b/getting_started/setup_vm/roles/ccf_build/tasks/install.yml
@@ -32,7 +32,7 @@
 
 - name: Build mbedtls
   shell: |
-    cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_POSITION_INDEPENDENT_CODE=ON ..
+    cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_POSITION_INDEPENDENT_CODE=ON ..
     make
   args:
     chdir: "{{ workspace }}/mbedtls-{{ mbedtls_dir }}/build"

